mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Koans.Mixfile do
   def project do
     [app: :elixir_koans,
      version: "0.0.1",
-     elixir: ">= 1.2.1 and <= 1.4.0",
+     elixir: ">= 1.2.1 and <= 1.4.1",
```

Comment:
I wonder if we should leave this more open ended to lower thrash in the config @felipesere @ukutaht?